src/main/java/duke/UI/UI.java

```diff
@@ -0,0 +1,67 @@
+package duke.UI;
+
+public class UI {
```

Comment:
While I get the abbreviation, I think that it should still follow the camelCase guideline
```suggestion
public class Ui {
```